client/jetpack-cloud/sections/agency-dashboard/hooks/use-toggle-activate-monitor.tsx

```diff
@@ -104,7 +104,11 @@ export default function useToggleActivateMonitor(
 			sites.forEach( ( site ) =>
 				requests.push( {
 					site,
-					mutation: toggleActivateMonitoring.mutateAsync( { siteId: site.blog_id, params } ),
+					mutation: toggleActivateMonitoring.mutateAsync( {
+						siteId: site.blog_id,
+						params,
+						hasJetpackPluginInstalled: site?.enabled_plugin_slugs.includes( 'jetpack' ),
```

Comment:
I think we should update this to something along these lines:

```
hasJetpackPluginInstalled: site?.enabled_plugin_slugs?.includes('jetpack') ?? false,
```

If the site doesn't have any plugins (it might happen) this value won't be an array.

Reply:
Agree. I have updated here: https://github.com/Automattic/wp-calypso/compare/644dde4846025f609a17cf9df303cc6b3a6e5da9..44c3f3e7004f5994c0047bc235649a013e25d7e2